great_ape_safe/great_ape_safe.py

```diff
@@ -192,6 +193,8 @@ def post_safe_tx(
             )
         if tenderly:
             self._generate_tenderly_simulation(receipt, safe_tx.safe_tx_gas)
```

Comment:
minor, but if this is moved to right before `if post` you dont need to scroll up to see the url

Reply:
moved it in bcb8d9f5c0b30e32ad5a2a147dc96a34d47e61dd

Reply:
what are the thoughts on auto-opening to save those clicks 🤔 , something like:

```
import webbrowser

webbrowser.open(url)
```

Reply:
i like it, even more convenient. if chosen to use it, i would put it into a try/except block so that the whole tx doesn't crash just because it isn't able to open up in a web browser for whatever reason. also would want to check compatibility between different OS's

Reply:
tbh i think it is a bit overkill. cmd+doubleclick is not that much effort. main consideration for me here is that a lot of times when running a script i might not be interested in seeing the tenderly. then having to close the browser might actually create unnecessary actions in the workflow.